packages/dd-trace/src/appsec/sdk/set_user.js

```diff
@@ -11,6 +11,12 @@ function setUserTags (user, rootSpan) {
   }
 }
 
+function setUserTagsSdk (user, rootSpan) {
```

Comment:
we can use this function in user_blocking.js too then

Reply:
actually i just realized, setUserTags() is not even used outside of this function anymore ? it's probably a relic of how automated user tracking used to use the sdk. We can probably just add the `rootSpan.setTag('_dd.appsec.user.collection_mode', 'sdk')` to the `setUserTags()` instead of making a new function no ?

Reply:
yes, I didn't realize that it was used only from sdk, I thought that it was used somewhere else, changing it.